tdrs-backend/tdpservice/settings/common.py

```diff
@@ -121,6 +121,7 @@ class Common(Configuration):
     AWS_S3_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
     AWS_STORAGE_BUCKET_NAME = os.getenv("AWS_BUCKET")
     AWS_REGION_NAME = os.getenv("AWS_REGION_NAME")
+    AWS_S3_REGION_NAME = os.getenv("AWS_REGION_NAME")
```

Comment:
I'm not sure why it was introduced with this PR and not #861 but without this additional setting attempts to upload to the cloud.gov S3 service tdp-datafiles-dev return the error:
```
An error occurred (InvalidAccessKeyId) when calling the PutObject operation: The AWS Access Key Id you provided does not exist in our records.
```
[Docs](https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#settings) for the django-storages package we use indicate that this is the correct setting name. With #971 I will update this to be pulled from the Cloud.gov `VCAP_SERVICES` and update any references to the old setting in auxiliary code that uses it.

Reply:
Nice catch.